adapter/test_slack.py

```diff
@@ -74,14 +76,36 @@ async def common_mock_request(self, mock_request, text):
                 "user_id": "U12345678",
                 "text": text,
                 "metadata" : {
-                    "event_type": "bot-slug",
+                    "event_type": "chat-data",
                     "event_payload": {
                         "bot_slug": "12"
                     }
                 }
             }
         )
         return mock_request
+    
+    def common_message_history(self):
```

Comment:
Let's make the names follow previous conventions. `message_history` or `chat_history`

Reply:
What naming conventions are you referring to?

Reply:
I mean like in some cases you refer to the "history" as `chat_history` and sometimes as `message_history`. I was thinking maybe we can be consistent in the naming. Either `chat_history` or `message_historty`. Unless of course they refer to different things.

Reply:
You're right, I didn’t notice the inconsistency. I appreciate you pointing this out.